packages/webpack/src/utils/postcss.js

```diff
@@ -79,11 +83,11 @@ export default class PostcssConfig {
           })
         },
 
-        // https://github.com/postcss/postcss-url
+        // https://github.com/csstools/postcss-plugins/plugin-packs/postcss-url
```

Comment:
I think the old url is still correct:

```suggestion
        // https://github.com/postcss/postcss-url
```